_posts/2023-07-06-release.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Release 2023.27
+author: Priya
+date: 2023-06-27
+category: Jekyll
+layout: post
+---
+
+### Enhancements:
+#### Data Passthrough Control (data-sync):
+Data Sync now provides enhanced capabilities for managing data connections and controlling the flow of data between different components. With these improvements, you can:
+###### a)Preview Panel: 
```

Comment:
* Gain access to data sync controls. 
* Access a wider range of dataset properties and view information.
* Access view controls directly from preview panel

Reply:
Updated